### PR TITLE
Add GISLook v1.0.2

### PR DESCRIPTION
**Pull request by jmunowitch**
#### Adding a new cask
- [X] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [X] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [X] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [X] Commit message includes cask’s name.
- [X] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses. **See Explanation Below**
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.

When running `brew cask style --fix {{cask_file}}`, I received this error:
https://github.com/caskroom/homebrew-cask/issues/20806

### CODE REVIEW

**Comment by vitorgalvao** [comment]
`uninstall delete:` is likely insufficient. `pkgutil:` will likely be needed. Have you followed the [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md)?

**Comment by jmunowitch** [comment]
The original documentation of the project specifics that removal of those files uninstalls the software. 

I updated the PR to use pkgutil instead.

File: Casks/gislook.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 'ac1e8a48bce216b17d573783a647cac2fb06a20c6c002ea45f7439341a0a10a1'
+
+  url 'http://cartography.oregonstate.edu/gislook/GISLook_GISMeta.pkg'
+  name 'GISLook'
+  homepage 'http://cartography.oregonstate.edu/GISLook.html'
+  license :other
+
+  pkg 'GISLook_GISMeta.pkg'
+
+  uninstall pkgutil: [
```
Comment (vitorgalvao) [inline Casks/gislook.rb]:
```ruby
 uninstall pkgutil: [
                      'ch.bernhardjenny.gislook.pkg',
                      'ch.bernhardjenny.gismeta.pkg',
                    ]
```

File: Casks/gislook.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask 'gislook' do
+  version '1.0.2'
```
Comment (vitorgalvao) [inline Casks/gislook.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by jmunowitch** [comment]
I updated the PR for version-less downloading and fixed the whitespace as requested.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 65684de69346599092ed6305e0c0ee1240b1a21c. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**